Detectors/TPC/base/src/TPCBaseLinkDef.h

```diff
@@ -9,6 +9,7 @@
 #pragma link C++ class AliceO2::TPC::LocalPosition2D;
 #pragma link C++ class AliceO2::TPC::LocalPosition3D;
```

Comment:
one general comment here: Shouldn't we put a '+' at the end of all pragma link lines? Otherwise, there will be no StreamerInfo included into the ROOT file.